Fix Localization

PurpleHato opened the pull request:
FIX: French Localization (Looks like it's okay after 5 small tests)
TWEAK: shard of agony -> stone of agony
TWEAK: odd poultice -> odd potion

PurpleHato committed:
FIX: French Loca

TWEAK: shard of agony -> stone of agony
TWEAK:  odd poultice -> odd potion

briaguya0 commented on soh/soh/Enhancements/randomizer/randomizer.cpp:
```diff
@@ -1352,7 +1352,7 @@ std::unordered_map<std::string, RandomizerGet> SpoilerfileGetNameToEnum = {
     { "Acheter: Missiles", RG_BUY_BOMBCHU_5 },
     { "Buy Deku Seeds (30)", RG_BUY_DEKU_SEEDS_30 },
     { "Acheter: Graines Mojo (30)", RG_BUY_DEKU_SEEDS_30 },
-    { "Sold Out", RG_SOLD_OUT },
```
just making sure we still have this in english too

PurpleHato replied:
Whoopsie! Done!

briaguya0 reviewed (changes requested):
overall it's looking great! super exciting stuff. as soon as the shard/poultice enum names are changed i think it's good to merge (assuming all the names are matching)

PurpleHato committed:
FIX: hint_list_item back to UTF8

PurpleHato committed:
TWEAK: 3DS to N64 names

ODD_POULTICE --> ODD_POTION
SHARD_OF_AGONY --> STONE_OF_AGONY

PurpleHato committed:
FIX: Sold Out

PurpleHato committed:
TWEAK: Even more fixes

PurpleHato committed:
Varios fix and tweaks

ADD: French Localization of Sphere (looks like it fixed the french issues)

TWEAK: More french fixes in various files

PurpleHato committed:
TWEAK: Localization comments

PurpleHato committed:
Tweak: some french hints fixes

PurpleHato committed:
FIX: French scrub (I was an idiot)

PurpleHato committed:
FIX: avoiding stranginess with the previous fix

PurpleHato committed:
FIX: FINAL FIX - After having a brain being dumb at 3AM

PurpleHato committed:
FIX: More UTF-8 fixes

PurpleHato committed:
FIX: Even more UTF8 fixes than before.... again

PurpleHato committed:
TWEAK: Made an whoopsie on the path while fixing UTF8

PurpleHato committed:
FIX: Bad translation

PurpleHato committed:
TWEAK: Agony and potion stuff again

briaguya0 reviewed (commented):
addresses https://github.com/briaguya-ai/rando-issue-tracker/issues/9 and https://github.com/briaguya-ai/rando-issue-tracker/issues/16
